examples/decoding/plot_decoding_ssd_csp_eeg.py

```diff
@@ -0,0 +1,237 @@
+"""
+.. _ex-decoding-ssd-csp-eeg:
+
+============================================================================
+Improving motor imagery decoding from EEG using Spatio Spectra Decomposition
+============================================================================
+
+The Spatio Spectra Decomposition (SSD) is a supervised spatial filtering
+algorithm capable of extracting components primarily related to the signal of
+interest. It can be used as a pre-processing step for the analysis of neuronal
+activity. In this example SSD will be used in the context of Motor Imagery
+decoding from EEG data. It will be applied before extracting features with
+the Common Spatial Patterns (CSP) method. A classifier will then be trained
+using the extracted features from the SSD + CSP-filtered signals. The impact in
+performance of using SSD before CSP will be shown here.
+
+:footcite:`NikulinEtAl2011`.
+"""
+# Authors: Victoria Peterson <victoriapeterson09@gmail.com>
+#
+# License: BSD (3-clause)
+
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+from sklearn.pipeline import Pipeline
+from sklearn.discriminant_analysis import LinearDiscriminantAnalysis
+from sklearn.model_selection import ShuffleSplit, cross_val_score
+
+from mne import (Epochs, pick_types, events_from_annotations,
+                 EvokedArray)
+from mne.channels import make_standard_montage
+from mne.io import concatenate_raws, read_raw_edf
+from mne.datasets import eegbci
+from mne.decoding import CSP
+from mne.decoding import SSD
+###############################################################################
+# Set parameters and read data
+
+# epochs from 0.5 to 2.5 s after the cue onset will be used
+tmin, tmax = 0.5, 2.5
+event_id = dict(left=2, right=3)  # Motor imagery: left vs right hand
+runs = [4, 8, 12]
+Subjects_set = [2, 7, 31, 34, 42, 56, 60, 62, 85, 100]
```

Comment:
```suggestion
subjects = [2, 7, 31, 34, 42, 56, 60, 62, 85, 100]
```
why? it's not a set it's more a list. Also in Python variables names (unless they are constants) should be in snake_case (no capital letters which are reserved for class names).

Reply:
oh, I was just thinking in "set" as a "dataset/subset", but sorry I forgot about the naming convention here!